### PR TITLE
Update to the licensing section

### PR DESCRIPTION
**Pull request by tsylvia88**
Saying that "A phone system license" is the only license needed is untrue.  You need to have a minimum of these 3 licenses. Microsoft Teams, Skype for Business Online Plan 2 and Microsoft 365 Phone System.   The admin portal wont even let you assign a Phone System license unless you have Teams and Skype for Business Online Plan 2, first.

### CODE REVIEW

File: Teams/sip-gateway-plan.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Teams users must have a phone number with PSTN calling enabled to use SIP Gatewa
 
 If you have a 3PIP or SIP device, you must have:
 
-- A license for Phone System (via E5 or a standalone license)
+- A license for Microsoft Teams, Skype for Business Online Plan 2 and Microsoft 365 Phone System (via E5 or a standalone licenses)
```
Comment (JohanFreelancer9) [inline Teams/sip-gateway-plan.md:59]:
```suggestion
- A license for Microsoft Teams, Skype for Business Online Plan 2, and Microsoft 365 Phone System (via E5 or standalone licenses)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tsylvia88, kindly commit the suggested change. Many thanks.

**Comment by tsylvia88** [comment]
@JohanFreelancer9  @scanum   Suggested changes look good.  Many thanks.

**Comment by JohanFreelancer9** [comment]
@tsylvia88, could you please commit the change? Thanks.

**Comment by tsylvia88** [comment]
> @tsylvia88, could you please commit the change? Thanks.

Changes looks good.

**Pull request merged**